data/sqlitebrowser

```diff
@@ -1,3 +1 @@
-https://github.com/sqlitebrowser/sqlitebrowser/releases/download/continuous/DB_Browser_for_SQLite--x86_64.AppImage
-# Very strage file name
-#
+https://github.com/sqlitebrowser/sqlitebrowser/releases/download/continuous/DB_Browser_for_SQLite-master-80-9ba1a2e-x86_64.AppImage
```

Comment:
Is the embedded string fragment `80-9ba1a2e` correct there?

Reply:
If `80-9ba1a2e` is the version string, then yes.